genesis/process/testdata/gasSchedule.toml

```diff
@@ -0,0 +1,588 @@
+[BuiltInCost]
```

Comment:
thi file is copied from left to rigth. duplicated 3 times.

Reply:
any ideea of how to create a gas model that will work in this case?

Reply:
you have a generate gasModel function in integration tests